modules/ROOT/pages/_partials/upgrade-tool.adoc

```diff
@@ -0,0 +1,61 @@
+//This content is reused in:
+//mule-upgrade-tool.adoc
+
+// Before You Begin
+// tag::BeforeYouBegin[]
+
+* A currently operational Mule instance running version 4.1.1 or later, in _stopped_ status to prepare for the upgrade.
++
+When updating your current Mule instance to a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022. If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made.
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory, that is, your old or existing Mule instance. 
++
+To download the tool, log in to https://help.mulesoft.com/s/[Help Center^]. Under the Support menu, select *Downloads > Mule Runtime*.
++
+An example of the file name is `mule-runtime-upgrade-tool-1.1.0.zip`.
```

Comment:
I thought we were recommending that they always use the latest version of the tool, which (eventually) might not be the one in the /tools subdir.

Reply:
We will leave this example as is for Mule 4.4 branch. And we will add a modification for Mule 4.5 branch as discussed with Julio.